apps/rush-lib/src/utilities/Npm.ts

```diff
@@ -8,12 +8,13 @@ export class Npm {
   public static publishedVersions(
     packageName: string,
     cwd: string,
-    env: { [key: string]: string | undefined }
+    env: { [key: string]: string | undefined },
+    extraArgs: string = ''
   ): string[] {
     const versions: string[] = [];
     try {
       const packageTime: string = Utilities.executeCommandAndCaptureOutput('npm',
-        `view ${packageName} time --json`.split(' '),
+        `view ${packageName} time --json ${extraArgs}`.trim().split(' '),
```

Comment:
>split [](start = 62, length = 5)

This is nasty. This should be initialized as an array instead of generating a string and then splitting it up.

Reply:
Good point...